src/EnergyPlus/VentilatedSlab.hh

```diff
@@ -164,8 +214,8 @@ namespace VentilatedSlab {
         int CCoil_Index;
         std::string CCoilPlantName; // name of cooling coil (child<=CoilSystem:Cooling:Water:HeatExchangerAssisted)
         std::string CCoilPlantType; // type of cooling coil (child<=CoilSystem:Cooling:Water:HeatExchangerAssisted)
-        DataPlant::PlantEquipmentType CoolingCoilType;
-        int CCoilType; // type of cooling coil:
+        DataPlant::PlantEquipmentType coolingCoilType;
```

Comment:
Why do we need both of these coil types?

Reply:
🤦‍♂️  I was afraid it would be something like this. We'll have to clean up later separately. Oh EnergyPlus ...